massgov/mayflower
# DP-8725: responsive tables with horizontal scroll

## Description
The addition of horizontal scroll to responsive tables

## Related Issue / Ticket

- [DP-8725](https://jira.mass.gov/browse/DP-8725)

## Steps to Test

Check the following URLs on feature1 in multiple browsers to confirm the desired functionality: 
* On this one check that the fees table matches production, but the table in the "More info" section is now a responsive table:
    * http://massgovfeature1.prod.acquia-sites.com/how-to/renew-your-nursing-license (feature1)
    * https://www.mass.gov/how-to/renew-your-nursing-license (production)
* This one has multiple tables: http://massgovfeature1.prod.acquia-sites.com/service-details/fruits-and-vegetables-inspection
    * Note that neither table actually has a thead even though the top one appears like it does.  The th cells in the top table are in the tbody.
    * Confirm on these tables that the scrolling works separately for all interactions.
* Here are more examples with various tables:
    * http://massgovfeature1.prod.acquia-sites.com/service-details/2018-massdot-contract-price-adjustments
    * http://massgovfeature1.prod.acquia-sites.com/service-details/pesticide-exams-rosters-and-results
    * http://massgovfeature1.prod.acquia-sites.com/service-details/encompass-insurance-company-of-ma-installment-payment-plans
    * http://massgovfeature1.prod.acquia-sites.com/service-details/learn-about-massachusetts-tax-rates
    * http://massgovfeature1.prod.acquia-sites.com/service-details/state-transportation-building-operations-and-contacts
    * http://massgovfeature1.prod.acquia-sites.com/service-details/view-tax-treatment-of-retirement-plan-contributions-and-distributions
    * http://massgovfeature1.prod.acquia-sites.com/service-details/basic-hunter-education-course-offerings
    * http://massgovfeature1.prod.acquia-sites.com/service-details/october-2017-results
    * http://massgovfeature1.prod.acquia-sites.com/service-details/primary-care-clinician-pcc-plan-for-masshealth-members



## Screenshots
